app/models/metadata.py

```diff
@@ -25,10 +25,36 @@ class OrganisationConfig(BaseModel):
     count_by_category: Mapping[str, int]
 
 
+class CorpusConfig(BaseModel):
```

Comment:
Should we be moving this under a corpus or config entity file rather than leaving this in metadata (DDD)??